src/Covid19Dashboard/WorldMapChart/index.jsx

```diff
@@ -192,7 +192,8 @@ class WorldMapChart extends React.Component {
 
       // density map data contains fips and iso3;
       // choropleth map data contains FIPS and iso_a3.
-      const fips = `${feature.properties.FIPS || feature.properties.fips}`;
+      let fips = feature.properties.FIPS || feature.properties.fips;
+      fips = fips ? `${fips}` : fips; // convert to string if not undefined
```

Comment:
this will evaluate to false if vale of fips is 0, is this acceptable?

Reply:
can use 
`typeof fips !== 'undefined'`

Reply:
fips 0 doesn't exist afaik but i'll update it to be safe